kind/goss/goss.yaml

```diff
@@ -0,0 +1,9 @@
+file:
+  /usr/local/bin/kind:
+    exists: true
+    mode: "0711"
```

Comment:
@s10myk4 
Add a test for the docker command.

Reply:
Fixed.
315d172